src/dealDashboard/dealDiscussion/discussionThread/discussionThread.ts

```diff
@@ -272,6 +276,27 @@ export class DiscussionThread {
         ],
         this.replyToComment?._id || "",
       );
+
+      if (newComment) {
+        this.threadComments.push({
+          _id: newComment._id,
+          text: newComment.text,
+          author: newComment.author,
+          authorENS: newComment.authorENS,
+          metadata: newComment.metadata,
+          replyTo: newComment.replyTo,
+          upvotes: newComment.upvotes,
+          downvotes: newComment.downvotes,
+          createdOn: newComment.createdOn,
+        });
```

Comment:
possible?:

1. 
```suggestion
        this.threadComments.push(newComment);
```

or

2.
```suggestion
        this.threadComments.push({ ...newComment });
```